lib/contributionreward.js

```diff
@@ -158,11 +177,16 @@ class ContributionReward extends ExtendTruffleContract(SolidityContributionRewar
   }
 
   async setParams(params) {
-    return await this._setParameters(params.orgNativeTokenFee, params.schemeNativeTokenFee, params.voteParametersHash, params.votingMachine);
+    return await this._setParameters(
+      params.orgNativeTokenFee,
+      params.schemeNativeTokenFee,
+      params.voteParametersHash,
+      params.votingMachine
+    );
   }
 
   getDefaultPermissions(overrideValue) {
-    return overrideValue || '0x00000001';
+    return overrideValue || "0x00000001";
```

Comment:
It will be nice to have meaning full permission..
which can be defined somewhere in a general header file.
e.g REGISTERED = "0x00000001".
so it will looks like return overrideValue || REGISTERED;

by the way , please note that permission 0x00000000 is same as permission 0x00000001 as at the moment this scheme is register to the controller ..this bit is set automatically .